controllers/storagecluster/cephrbdmirrors.go

```diff
@@ -0,0 +1,137 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+
+	ocsv1 "github.com/openshift/ocs-operator/api/v1"
+	"github.com/openshift/ocs-operator/controllers/defaults"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type ocsCephRbdMirrors struct{}
+
+func (r *StorageClusterReconciler) fetchCephRbdMirrorInstance(cephRbdMirror *cephv1.CephRBDMirror) (*cephv1.CephRBDMirror, error) {
+	existing := &cephv1.CephRBDMirror{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: cephRbdMirror.Name, Namespace: cephRbdMirror.Namespace}, existing)
+	return existing, err
+}
+
+func (r *StorageClusterReconciler) deleteCephRbdMirrorInstance(cephRbdMirrors []*cephv1.CephRBDMirror, flowName string) error {
```

Comment:
let's remove the notation of flow name. Why does it matter?

Reply:
I saw all other resource reconcilers are using the prefix "uninstall" in the log inside delete flow. I thought the user wants to see delete related logs separately for easy debugging. Since I am calling the delete function from creating and delete functions, I passing the flow name as parameter to log.

As per above comment(https://github.com/openshift/ocs-operator/pull/1301#discussion_r693757549), I will remove the flow name param.